testutil/payloads.go

```diff
@@ -230,6 +232,23 @@ const RestoreYaml = `restore:
   workload_agent_uuid: ` + AgentUUID + `
 `
 
+// CNCITunnelID is a gre tunnel ID derived from the tenant UUID
+var CNCITunnelID = crc32.ChecksumIEEE([]byte(TenantUUID))
```

Comment:
What happens if there is a clash?

Reply:
The probability of a collision is small for the likely number of tenants. On top of that, I think you'd have to have the same tunnelID for the same tunnel source and destination for it to be an issue. We should check with manohar to see if this is something we really need to be concerned about.

Reply:
@mcastelino can you check my reasoning here?

Reply:
@rbradford great question. Luckily for us, here the tunnel traffic is only examined once it lands inside the CNCI. So even if we end up with a collusion it will not be an issue as it will not conflict within the CNCI. Nothing on the wire looks at the GRE tunnel info, so that helps us.